test/test_model.py

```diff
@@ -90,6 +95,14 @@ def test_get_relation(self):
         with self.assertRaises(op.model.TooManyRelatedApps):
             self.model.get_relation('db2')
 
+    def test_remote_units_not_local(self):
+        for u in self.model.get_relation('db1').units:
+            self.assertFalse(u.is_local)
+
+    def test_local_unit_is_local(self):
+        self.assertTrue(self.model.unit.is_local)
+        self.assertTrue(self.model.unit.app.is_local)
```

Comment:
No app is ever tested for being not local. Let's please have a single test which checks both conditions for both types.